### PR TITLE
Add login router styling

### PR DESCRIPTION
**Pull request by robinschaaf**
Note: some of these differences should resolve once PR #29 is in.
Styling updates:
-  Add fonts and iconsets - Open Sans, Zooniverse Icons, Font Awesome
-  Update Android assets/images and app label (not the full name - just the app label only for now)
-  Update the iOS launch screen
-  Add spinner overlay
-  Add high level category page (and rename to Disciplines to be consistent with PFE)
  App additions:
-  Add login screen
-  Add router
-  Add simple store for storing user info locally

### CODE REVIEW

**Comment by robinschaaf** [comment]
I'll get these merge conflicts cleaned up and push back up

File: src/components/Discipline.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+  },
+});
+
+Discipline.propTypes = {
+  icon: React.PropTypes.string.isRequired,
+  title: React.PropTypes.string.isRequired,
+  tag: React.PropTypes.string.isRequired,
+  color: React.PropTypes.string.isRequired
+}
+
```
Comment (wgranger) [inline src/components/Discipline.js]: Is there a reason these propTypes are defined here rather than above in the Discipline class?

**Comment by wgranger** [comment]
Is it common practice in React Native to put styles in file with the component? Just curious, as I've noticed that's how we've been doing it.

File: src/components/ProjectList.js [hunk 2]
```diff
@@ -66,7 +41,7 @@ class ProjectList extends React.Component {
   }
 
   getProjects(){
-    apiClient.type('projects').get({id: MOBILE_PROJECTS, cards: true, sort: 'display_name'})
+    apiClient.type('projects').get({id: MOBILE_PROJECTS, cards: true, tags: this.props.tag, sort: 'display_name'})
```
Comment (wgranger) [inline src/components/ProjectList.js]: What does this `tags` key request? I'm unfamiliar with this.

Comment (robinschaaf) [inline src/components/ProjectList.js]: TBH, I looked up how the Projects/Discipline filter works in PFE, and that is how they query / filter on Disciplines in the project-card-list component.  I'm not sure why it has that name, but I'm guessing that disciplines are "tagged" on the project.

**Comment by robinschaaf** [comment]
Re: styles in component:  That's how all of the sample code and sample projects I've seen do it handle it.  It is possible to have them pulled out in a separate file, but I've personally found it easier to have it contained together.

**Pull request merged**